libraries/Microsoft.Bot.StreamingExtensions/Payloads/Assemblers/ContentStreamAssembler.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Bot.StreamingExtensions.Payloads
+{
+    internal class ContentStreamAssembler : PayloadAssembler
+    {
```

Comment:
There *IS* a class in here called "ContentStream". What's it relationship with this class? This class doesn't seem to do anything with that, and has no obvious relationship.